### PR TITLE
upgrade macos

### PR DESCRIPTION
**Pull request by QuanluZhang**

### CODE REVIEW

File: test/pipelines/pipelines-it-installation.yml [hunk 1]
```diff
@@ -16,7 +16,7 @@ jobs:
 
 - job: 'pip_install_macOS_python36'
   pool:
-    vmImage: 'macOS-10.13'
+    vmImage: 'macOS-latest'
   strategy:
     matrix:
```
Comment (chicm-ms) [inline test/pipelines/pipelines-it-installation.yml:21]: the `strategy` section can be removed,  it does not take effect as we expected.
The actual default python version depends on the image.

**Review by ultmaster — APPROVED** [review]

File: test/pipelines/pipelines-it-installation.yml (hunk 1 above)
Comment (QuanluZhang) [reply test/pipelines/pipelines-it-installation.yml]: updated, thanks!

**Review by chicm-ms — APPROVED** [review]

File: test/pipelines/pipelines-it-installation.yml [hunk 2]
```diff
@@ -16,11 +16,7 @@ jobs:
```
Comment (chicm-ms) [inline test/pipelines/pipelines-it-installation.yml:16]: better to remove the `python36` in job name

Comment (QuanluZhang) [reply test/pipelines/pipelines-it-installation.yml:16]: updated

**Pull request merged**